lib/hashie/extensions/coercion/dry_types.rb

```diff
@@ -0,0 +1,37 @@
+require 'dry-types'
+
+module Hashie
+  module Extensions
+    module Coercion
+      module DryTypes
+        module Types
+          include Dry::Types.module
```

Comment:
For consistency and protection against accidental problems later, I think fully qualifying this as `::Dry::Types.module` is a good idea.